docs/android/setup.md

```diff
@@ -24,10 +24,18 @@ and the following code:
 </receiver>
 ```
 
-Add the following NuGet package:
-
+Click on “**Project>Add NuGet Packages**” and find and add the following NuGet package:
+```
 - Xamarin.Android.Support.Compat v7 (min version: 26.0.2-rc1)
 
+- Xamarin.Firebase.Messaging (min version: 57.1104.0-beta1)
+
+- Xamarin.GooglePlayServices.Location (min version: 57.1104.0-beta1)
+```
+
+**WARNING**
+In either mode, **Debug** and **Release**, go to “**Options>Android Application**” and enable Multi-Dex mode.
```

Comment:
In either mode, **Debug** and **Release**, make sure you have enabled Multi-Dex mode in “**Options>Android Application**”.